scripts/core/registry.yml

```diff
@@ -367,9 +367,18 @@ etors:
 - name: COMMAND_BUFFER_FINALIZE_EXP
   desc: Enumerator for $xCommandBufferFinalizeExp
   value: '123'
+- name: COMMAND_BUFFER_APPEND_MEMBUFFER_WRITE_EXP
+  desc: Enumerator for $xCommandBufferAppendMembufferWriteExp
+  value: '124'
```

Comment:
Looks like these ID's were generated before we updated the script to only append after the highest ID value.

If you could undo all the changes in this file, then rerun the `generate` target and push the new version I'd appreciate it. All of these entry-points *should* be grouped together at the end of the file.

Reply:
Thanks. The re-generation solved the issue. :+1: